### ISSUE TITLE
Many error console

### ISSUE THREAD

**Issue by Mick43** [FlyingPikachu/Quests#26]
With the last build and all file regenate, i have many error when start server:

[Server thread/ERROR]: Could not pass event PlayerJoinEvent to Quests v2.2.2
org.bukkit.event.EventException
at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:305) ~[spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at net.minecraft.server.v1_8_R1.PlayerList.onPlayerJoin(PlayerList.java:272) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at net.minecraft.server.v1_8_R1.PlayerList.a(PlayerList.java:156) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at net.minecraft.server.v1_8_R1.LoginListener.b(LoginListener.java:109) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at net.minecraft.server.v1_8_R1.LoginListener.c(LoginListener.java:41) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at net.minecraft.server.v1_8_R1.NetworkManager.a(NetworkManager.java:159) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at net.minecraft.server.v1_8_R1.ServerConnection.c(ServerConnection.java:82) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at net.minecraft.server.v1_8_R1.MinecraftServer.z(MinecraftServer.java:800) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at net.minecraft.server.v1_8_R1.DedicatedServer.z(DedicatedServer.java:316) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at net.minecraft.server.v1_8_R1.MinecraftServer.y(MinecraftServer.java:634) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at net.minecraft.server.v1_8_R1.MinecraftServer.run(MinecraftServer.java:537) [spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
at java.lang.Thread.run(Thread.java:722) [?:1.7.0]
Caused by: java.lang.NoSuchMethodError: java.util.concurrent.ConcurrentHashMap.keySet()Ljava/util/concurrent/ConcurrentHashMap$KeySetView;
at me.blackvein.quests.PlayerListener.onPlayerJoin(PlayerListener.java:838) ~[?:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0]
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[?:1.7.0]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.7.0]
at java.lang.reflect.Method.invoke(Method.java:601) ~[?:1.7.0]
at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:301) ~[spigot-1.8-R0.1-SNAPSHOT.jar:git-Spigot-35348a5-ee6d0fa]
... 14 more

**Comment by Formula350** [comment]
Do any of your quests names have a Period -> ** . ** <-- in them? If so, remove that from the quest name, and then you will have to go into the Data folder and either: Delete the player's file **or** edit the file so that the quest name under "currentQuests:" is now the same as what the quest's name has been changed to.

What happened for me was I had a quest called: Bow of Stars Pt. 1
In player Data files it was put in as: Bow of Stars Pt ' 1'
I changed the quest's name inside quests.yml to: Bow of Stars Pt1
I edited Data files by hand to fix anyone who had that quest so it now matched quests.yml

Hope that helps :\
